Servers/UI/OJS.Servers.Ui/ClientApp/src/components/contests/contest/Contest.tsx

```diff
@@ -86,9 +89,7 @@ const Contest = () => {
 
             const currentSeconds = remainingTimeInMilliseconds / 1000;
 
-            return (
-                <Countdown renderRemainingTime={renderCountdown} duration={currentSeconds} metric={Metric.seconds} />
-            );
+            return <Countdown renderRemainingTime={renderCountdown} duration={currentSeconds} metric={Metric.seconds} />;
```

Comment:
Is this `eslint` automatic fix, or manual?

Reply:
I refactored it.